src/pages/Card/_components/SidebarFilter.tsx

```diff
@@ -16,25 +21,61 @@ function SidebarFilter() {
     CHECK_BOX_DATA.map((category) => Array(category.checkboxes.length).fill(false)),
   );
```

Comment:
이렇게 초기값이 길어질 떄는 따로 분리하는 방법도 추천드릴게요~~~

```suggestion

const initialActiveCategories =  Array(CHECK_BOX_DATA.length).fill(false),
const initailCheckBoxStates =  CHECK_BOX_DATA.map((category) => Array(category.checkboxes.length).fill(false))

function SidebarFilter({ onFilterChange, onAllCheck }: SidebarFilterProps) {
  const [isAllChecked, setIsAllChecked] = useState(false);
  const [activeCategories, setActiveCategories] =  useState( initialActiveCategories );
  const [checkboxStates, setCheckboxStates] = useState( initailCheckBoxStates);
  
```

Reply:
오오 분리하면 더 코드가 가독성 있겠네요! 감사해요!!